omero/conf.py

```diff
@@ -82,9 +82,13 @@
     'commit' : (omero_github_root + 'commit/%s', ''),
     'omedocs' : (doc_github_root + '%s', ''),
     # API links
-    'javadoc' : (downloads_root + '/latest/omero5.1/api/%s', ''),
-    'pythondoc' : (downloads_root + '/latest/omero5.1/api/python/%s', ''),
-    # Virtual Appliance downloads
+    'javadoc' : (downloads_root + '/latest/omero/api/%s', ''),
```

Comment:
I assume these will be replaced by versioned latest redirects when we have an intermediate tag